shoop/testing/factories.py

```diff
@@ -449,8 +451,8 @@ def create_empty_order(prices_include_tax=False, shop=None):
         shop=(shop or get_shop(prices_include_tax=prices_include_tax)),
         payment_method=get_default_payment_method(),
         shipping_method=get_default_shipping_method(),
-        billing_address=get_address(name="Mony Doge"),
-        shipping_address=get_address(name="Shippy Doge"),
+        billing_address=get_address(immutable=True, name="Mony Doge"),
+        shipping_address=get_address(immutable=True, name="Shippy Doge"),
```

Comment:
actually, this could as easily be `get_address(name="Shippy Doge").immutable()`. So why bother with the immutable argument to get_address? Rather just make it always return MutableAddress.